src/components/Hero.tsx

```diff
@@ -0,0 +1,116 @@
+import React, { useEffect, useState } from 'react'
+import key from '../assets/key.svg'
+import cmdK from '../assets/cmdK.svg'
+import starIcon from '../assets/starIcon.svg'
+import issueIcon from '../assets/issueIcon.svg'
+import { fetchWithSearch } from "../lib/supabase";
+import useDebounce from '../hooks/useDebounce'
+import humanizeNumber from "../lib/humanizeNumber";
+
+type PostResult = {
+  full_name: string;
+  user_id: number;
+  description: string;
+  stars: number;
+  issues: string;
+  contributions: { url: string; contributor: string; last_merged_at: string }[];
+};
+
+const Hero = () => {
+  const [searchTerm, setSearchTerm] = useState<string>("")
+  const debouncedSearchTerm: string = useDebounce<string>(searchTerm, 500);
+  const [results, setResults] = useState<PostResult[]>([]);
+  const [hasFocus, setFocus] = useState<boolean>(false);
+  
+  
+  useEffect(() => {
+    if (debouncedSearchTerm) {
+
+      fetchWithSearch("stars", 3, debouncedSearchTerm).then((results) => {
+        console.log(results)
+        setResults(results as unknown as []);
+      });
+    } else {
+      setResults([]);
+    }
+  }, [debouncedSearchTerm])
+  
+
+  return (
+    <div className='flex flex-col my-[95px] items-center'>
+        <div>
+          <h1 className='font-Lexend text-[48px] text-center text-lightSlate leading-[52.8px] tracking-[-6%] '>
+            Find <span className='bg-gradient-to-r from-gradFirst via-gradMiddle to-gradLast bg-clip-text text-transparent' >Open-Source Repositories</span> <br/> to  contribute today
+          </h1>
+        </div>
+        <div className='mt-[45px] px-[15px] gap-x-[10px] py-[10px] justify-between bg-white shadow-[0_35px_60px_-15px_rgba(0,0,0,0.3)] rounded-[16px] min-w-[422px]  flex'>
+          <img src={key} alt="key" />
```

Comment:
I'll provide a new icon, because this "key" icon doesn't make much sense... 

The Figma file is updated:

<img width="568" alt="Screen Shot 2022-07-20 at 7 27 32 PM" src="https://user-images.githubusercontent.com/101464991/180093189-982a8c42-5b33-4660-ae0f-1873d396435b.png">